README.md

```diff
@@ -1,15 +1,13 @@
 [![Build Status](https://semaphoreci.com/api/v1/calico/libcalico-go/branches/master/badge.svg)](https://semaphoreci.com/calico/libcalico-go)[![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org) [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 
 # libcalico-go
-This repositiory contains Calico's Go components:
+This repositiory contains Calico's shared Go library, `libcalico`, which can be imported as 
```

Comment:
I don't see any sense or place (other than this PR) in which 'libcalico-go' is named 'libcalico', and given that we have another (different!) thing named 'libcalico', I don't think it's a good idea to start promoting 'libcalico' as the name for this.  Just call it 'libcalico-go', surely?